test/unit/app/utils/rewards/test_processor.py

```diff
@@ -0,0 +1,23 @@
+from app.server.utils.rewards.processor import get_user_by_id
```

Comment:
I can't really see any test that checks whether the disbursement amounts are correct? Are those tests somewhere else?